packages/wallets/src/components/Base/Tooltip/Tooltip.tsx

```diff
@@ -3,16 +3,23 @@ import './Tooltip.scss';
 
 type TProps = {
     alignment?: 'bottom' | 'left' | 'right' | 'top';
+    className?: string;
     isVisible: boolean;
     message: string;
 };
 
-const Tooltip: React.FC<React.PropsWithChildren<TProps>> = ({ alignment = 'left', children, isVisible, message }) => {
+const Tooltip: React.FC<React.PropsWithChildren<TProps>> = ({
+    alignment = 'left',
+    children,
+    className,
+    isVisible,
+    message,
+}) => {
     return (
         <div className={`wallets-tooltip`}>
             {children}
             {isVisible && (
-                <div className={`wallets-tooltip__content wallets-tooltip__content--${alignment}`}>
+                <div className={`wallets-tooltip__content wallets-tooltip__content--${alignment} ${className}`}>
```

Comment:
@arshad-rao-deriv just one small thing for the future. If you leave it like this and don't pass className you'll see `undefined` in the list of classes of the current div :) pretty sure we don't have class with name undefined and it doesn't work anything, but it's just a little issue. To prevent this you can use classNames or write like this `${className ?? ''}`

Reply:
Updated.